packages/ssx-sdk/src/core.ts

```diff
@@ -127,11 +132,11 @@ export class SSXConnected {
     }
   }
 
-  public async ssxServerNonce(params: Record<string, any>): Promise<{ nonce: string }> {
+  public async ssxServerNonce(params: Record<string, any>): Promise<string> {
```

Comment:
This function returns `undefined` if `this.api` is falsy - is that the intended behavior?

Reply:
Also see my note in the general review about `try...catch`!

Reply:
Yes, [this is intended](https://github.com/spruceid/ssx/pull/10/files/fe82e7fed678495aab6986ae7da4d4bf7d27308e#diff-92023938c9c08f9a708ddadca984236b869fbfe8b4622000e0fd4d4e3f859220R194)!